test/api/two/two_peers.cpp

```diff
@@ -109,7 +109,7 @@ TEST(api, test_two_peers)
     ASSERT_EQ(4002, port);
 
     const char* version = dataZero["version"];
-    ASSERT_STREQ("2.0.0", version);
+    ASSERT_STREQ("2.0.11", version);
```

Comment:
Core version updated to `2.0.12`.
https://github.com/ArkEcosystem/core/releases/tag/2.0.12

Probably no way to do `ASSERT_GT`, otherwise mocks will prevent us from constantly having to revise.

Reply:
We could do a ASSERT_GT concept on this, it would just be a little extra code.

I would vote to just change the line to ASSERT_STRNEQ("", version);.

When mocks come in, we can do a proper validation for format (e.g. x.y.z).

Reply:
!=`“”` works for me 👍

That way we don’t have to keep running back to it before mocks are ready.

Thank you, @ciband